### DISCUSSION TITLE
Ways to read the default color after reboot / suspend

### DISCUSSION THREAD

**Discussion by MischaBaars** [liquidctl/liquidctl#545] [Q&A]
Hello,

I just had a look at liquidctl because reading the RGB control chips internal memory seems to return the same chunk of memory for all possible inputs.

So I wondered if liquidctl would do a better job in reading the current state of the RGB (Gigabyte/ITE 5702) controller, but it appears there is no command line option to read the current state. I would like to start out from the default color of the cooler and fans if possible.

Any thoughts on this?

Best regards,
Mischa Baars.

**Comment by jonasmalacofilho** [comment]
We don't know how to that, protocol wise, with that device.  If you can figure that part out, we can try to implement it.

**Comment by MischaBaars** [comment]
Oh, I'm sorry. I don't use Microsoft Windows, so I can't run the Gigabyte Fusion software.

I was trying to obtain a manual of the IT5702-GIGABYTE somewhere instead, but without luck so far.
